test/ex_machina/ecto_strategy_test.exs

```diff
@@ -53,4 +53,10 @@ defmodule ExMachina.EctoStrategyTest do
       TestFactory.insert(:user, name: "Maximus") |> TestFactory.insert
     end
   end
+
+  test "create/1 raises helpful error message directing use of insert/1" do
```

Comment:
Yeah I can see why that might make sense, but since the error on ExMachina I think it should go with the other tests
> On May 11, 2016, at 10:20 AM, David Kuhta <notifications@github.com> wrote:
> 
> In test/ex_machina/ecto_strategy_test.exs <https://github.com/thoughtbot/ex_machina/pull/129#discussion_r62856528>:
> 
> > @@ -53,4 +53,10 @@ defmodule ExMachina.EctoStrategyTest do
> >        TestFactory.insert(:user, name: "Maximus") |> TestFactory.insert
> >      end
> >    end
> > +
> > +  test "create/1 raises helpful error message directing use of insert/1" do
> Ok I can move it, I had only put it there b/c all the insert/1 tests were in ecto_strategy_test. While the insert/2, insert_pair/2, insert_list/3 tests were in ex_machina_test.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/thoughtbot/ex_machina/pull/129/files/8ee1e519cc97978edb17218bc2a7eb75046dff0d#r62856528>